### ISSUE TITLE
Can't get --extensions option to work when using CLI

### ISSUE THREAD

**Issue by axelboc** [BrowserSync/browser-sync#1220]
### Issue details

I can't seem to get vanity URLs (i.e. optional `html` extension) to work when running Browserify from the command line.

### Steps to reproduce/test case

1. Place two files, `index.html` and `foo.html`, inside a folder called `public`
2. Add the following anchor link to `index.html`: `<a href="/foo">foo</a>`
3. From the parent directory of `public`, run `browser-sync start --server "public" --extensions "html"`
4. `localhost:3000` opens and serves `index.html` automatically; select the link
5. Browser (in my case Firefox) shows: "Cannot GET /foo"

### Please specify which version of Browsersync, node and npm you're running

- Browsersync [  2.17.0  ]
- Node            [  6.5.0  ]
- Npm             [  3.10.3  ]

### Affected platforms

- [ ] linux
- [x] windows
- [ ] OS X
- [ ] freebsd
- [ ] solaris
- [ ] other _(please specify which)_

### Browsersync use-case

- [ ] API
- [ ] Gulp
- [ ] Grunt
- [x] CLI

### If CLI, please paste the entire command below

I tried:

```
browser-sync start --server "public" --extensions "html"
browser-sync start --server "public" --extensions ".html"
```

**Comment by shakyShane** [comment]
Fixed in v2.17.5

Thank you reporting the issue :)

**Comment by axelboc** [comment]
Awesome! Thanks 😄
